File: java/java-lambda/src/main/java/com/hoonjin/study/java/optional/model/Order.java
package com.hoonjin.study.java.optional.model;

import lombok.Getter;

@Getter
public class Order {

    private Long id;
    private Delivery delivery;

    public Order(Long id, Delivery delivery) {
        this.id = id;
        this.delivery = delivery;
    }
}

File: kotlin/kotlin-commerce/core/core-api/src/main/kotlin/com/hoonjin/study/kotlin/commerce/core/domain/Order.kt
package com.hoonjin.study.kotlin.commerce.core.domain

import com.hoonjin.study.kotlin.commerce.core.enums.OrderState
import java.math.BigDecimal

data class Order(
    val id: Long,
    val key: String,
    val name: String,
    val userId: Long,
    val totalPrice: BigDecimal,
    val state: OrderState,
    val items: List<OrderItem>,
)
